regions/io/tests/test_ds9_language.py

```diff
@@ -1,17 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
-import distutils.version as v
+
+import os
+import distutils.version as vers
+
 from numpy.testing import assert_allclose
+
 from astropy.utils.data import get_pkg_data_filename, get_pkg_data_filenames
 from astropy.tests.helper import pytest
 import astropy.version as astrov
 from astropy.coordinates import Angle, SkyCoord
+from astropy.tests.helper import catch_warnings
+from astropy.utils.exceptions import AstropyUserWarning
+
 from ...shapes.circle import CircleSkyRegion
-from ..read_ds9 import read_ds9, ds9_string_to_objects
+from ..read_ds9 import read_ds9, ds9_string_to_objects, global_parser
 from ..write_ds9 import write_ds9, ds9_objects_to_string
 
-_ASTROPY_MINVERSION = v.LooseVersion('1.1')
-_ASTROPY_VERSION = v.LooseVersion(astrov.version)
+_ASTROPY_MINVERSION = vers.LooseVersion('1.1')
```

Comment:
We are working towards putting this as `astropy.regions` in the core package.
So there's no need to support old Astropy versions.

At the moment `regions` has to work with Astropy 1.2, and we can switch to 1.3 as soon as that is out.

So remove this `xfail` for Astropy < 1.1 here.